cmd/gateway/s3/gateway-s3.go

```diff
@@ -137,13 +164,13 @@ func newS3(url, accessKey, secretKey string) (*miniogo.Core, error) {
 	if err != nil {
 		return nil, err
 	}
-
-	if _, err = clnt.BucketExists("probe-bucket-sign"); err != nil {
+	probeBucketName := randString(60, rand.NewSource(time.Now().UnixNano()), "probe-bucket-sign-")
+	if _, err = clnt.BucketExists(probeBucketName); err != nil {
 		clnt, err = miniogo.NewV2(endpoint, accessKey, secretKey, secure)
 		if err != nil {
 			return nil, err
 		}
-		if _, err = clnt.BucketExists("probe-bucket-sign"); err != nil {
+		if _, err = clnt.BucketExists(probeBucketName); err != nil {
```

Comment:
shouldn't we delete the bucket after probe is done?

Reply:
or may be while gateway shutdown

Reply:
We never create a bucket @nitisht

Reply:
> We never create a bucket

oops!